### ISSUE TITLE
Bump kind to v0.8.0

### ISSUE THREAD

**Issue by randomvariable** [kubernetes-sigs/cluster-api#2992]
**Detailed Description**

Upgrade to kind v0.8.x as per https://github.com/kubernetes-sigs/kind/releases/tag/v0.8.0

Requires API machinery v1.18.2, so will be dependent on a controller-runtime update occurring first..

**Anything else you would like to add:**

[Miscellaneous information that will assist in solving the issue.]

/kind feature

**Comment by cpanato** [comment]
I would like to help on this, can you point to me some path where we need to update? @randomvariable

**Comment by cpanato** [comment]
should we update the dependencies for 0.18.2 first? and then do this update?

**Comment by vincepri** [comment]
@cpanato we’ll need to wait for #2961 to be merged before we can update the kind version. Given that controller runtime v0.6 brings in breaking changes, we won’t be able to get this done until we can open master to v0.4

**Comment by vincepri** [comment]
Superseded by #3696 

/close
